tests/integration/test_debug_ssh.py

```diff
@@ -3,26 +3,28 @@
 
 """Integration tests for github-runner charm with ssh-debug integration."""
 import logging
-import typing
-import zipfile
-from io import BytesIO
+from datetime import datetime, timezone
 
-import requests
 from github.Branch import Branch
 from github.Repository import Repository
-from github.Workflow import Workflow
 from github.WorkflowRun import WorkflowRun
+from juju.application import Application
 
-from tests.integration.helpers import wait_for
+from tests.integration.charm_metrics_helpers import (
+    _get_job_logs,
+    dispatch_workflow,
+    get_workflow_runs,
+)
```

Comment:
Could you move the definition of these 3 imported functions to `helpers.py`? `charm_metrics_helpers.py` should only contain helpers related to the `charm_metrics` integration tests. Thank you very much!

Reply:
I've moved functions around to prevent circular imports, the dependencies are now in order of imports: helpers > charm_metrics_helpers > test module.

Thank you for the review!